source/application/IDEAS/IDEASAppController.cpp

```diff
@@ -156,7 +160,7 @@ void IDEASAppController::setupUserInterface()
 
 	mw_->insertHeading("XRF Detectors", 1);
 
-	ideasKETEKDetailedDetectorViewWithSave_ = new IDEASXRFDetailedDetectorViewWithSave(IDEASBeamline::ideas()->ketek());
+	KetekDetailedDetectorView *ideasKETEKDetailedDetectorViewWithSave_ = new KetekDetailedDetectorView(IDEASBeamline::ideas()->ketek());
```

Comment:
Hey @dleacock, you forgot the IDEAS prefix to the class name.  The views should be _IDEASKetekDetailedDetectorView_.  Could you go ahead and change these?  You'll need to make sure all the includes change, the class protection macro, and the .pri/.pro files.